deploy/helm/rawfile-localpv/values.yaml

```diff
@@ -6,6 +6,11 @@ logLevel: INFO
 # -- Format of the logs (json, pretty)
 logFormat: json
 
+# -- Reserved storage amount on each node (Supports percentage and size) [e.g. `25%` or `50GB` or `10MiB`]
```

Comment:
I think this to be made clear here that is storage reserved for the host and not for the rawfile itself